scripts/build.sh

```diff
@@ -6,7 +6,7 @@
 set -ev
 
 # Get the latest versions of packaging tools
-python3 -m pip install --upgrade pip setuptools wheel
+python3 -m pip install --user --upgrade pip setuptools wheel
```

Comment:
Note that this conflicts with virtualenvs:

```
ERROR: Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
```

Reply:
ah ok. I guess this is just a how-you-run-it kind of thing. For me the --user helps because I don't use a virtualenv, and I don't like to install these to the root environment.

But I'll try not to break workflows, I'll remove --user.